lib/decode_session_cookie.rb

```diff
@@ -0,0 +1,53 @@
+# decode_session_cookie.rb
+# ------------------------
+# The purpose of this script is to show that if I have the secret_key_base
+# and a cookie to an active Rails session, I can decrypt it and gain access
+# to key information about the user's session.
+
+require 'rubygems'
+require 'cgi'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.